CrossPlatformDesktopProject/CrossPlatformDesktopProject/Levels/Background.cs

```diff
@@ -103,5 +104,26 @@ public void DoorCheck(SpriteBatch sb, string[] adjacentRooms, Texture2D texture)
                 sb.Draw(texture, rightDoorDest, RoomTextureStorage.RIGHT_OPEN_DOOR, Color.White);
             }
         }
+        public List<Door> FindDoorColliders(string[] adjacentRooms, Map currentMap)
+        {
+            List<Door> doors = new List<Door>();
+            if (!adjacentRooms[0].Equals("-1"))
+            {
+                doors.Add(new Door(new Rectangle(0, topDoorDest.Center.Y, screenWidth, 1), currentMap, adjacentRooms[0]));
+            }
+            if (!adjacentRooms[1].Equals("-1"))
+            {
+                doors.Add(new Door(new Rectangle(0, bottomDoorDest.Center.Y, screenWidth, 1), currentMap, adjacentRooms[1]));
+            }
+            if (!adjacentRooms[2].Equals("-1"))
+            {
+                doors.Add(new Door(new Rectangle(leftDoorDest.Center.X, 0, 1, screenHeight), currentMap, adjacentRooms[2]));
+            }
+            if (!adjacentRooms[3].Equals("-1"))
+            {
+                doors.Add(new Door(new Rectangle(rightDoorDest.Center.X, 0, 1, screenHeight), currentMap, adjacentRooms[3]));
+            }
+            return doors;
+        }
```

Comment:
This & DoorCheck() could probably be cleaned up together, but we can worry about that during refactoring after the sprint.